### DISCUSSION TITLE
Problems with "add to desktop" link

### DISCUSSION THREAD

**Discussion by superfunk2000** [Lissy93/dashy#917] [Help and Support] [answered]
Hello Lissy,

I have my Dashy page saved to the desktop on my Android phone using "Add to Desktop" (MS Edge).
If I click on the link, the following page always appears first:

![Screenshot_20220922-080913_](https://user-images.githubusercontent.com/90634124/191747134-31b2aa23-37df-42af-b251-a42a93d6fc78.jpg)

I have to click on "Back Home" and wait about 10 seconds for Dashy to appear.

Why is that?

Many greetings
Joerg

**Comment by M0DY2002** [comment, marked as answer]
I had the same problem, found this solution:
https://dashy.to/docs/troubleshooting/#404-after-launch-from-mobile-home-screen

- change "routing mode" to "hash" (default is "history")

1. enter "interactive editor"
2. choose "edit app config"
3. change value of "routing mode" to "hash"
4. save 2x ;)

![image](https://user-images.githubusercontent.com/105606287/191933180-2783a916-b746-40af-8d8a-12e060d2e59b.png)

**Reply by superfunk2000** [reply]
Yes it was, thank you very much!

I entered it directly in config.yaml because my configuration is deactivated:
`  routingMode: "hash"`
